Demo/ImagePickerDemo/ImagePickerDemo/ViewController.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  ViewController.swift
+//  ImagePickerDemo
+//
+//  Created by Khoa Pham on 11/8/15.
+//  Copyright © 2015 Ramon Gilabert Llop. All rights reserved.
+//
+
+import Foundation
+import UIKit
+import ImagePicker
+
+class ViewController : UICollectionViewController, ImagePickerDelegate, UICollectionViewDelegateFlowLayout {
+  var images = [UIImage]()
+
+
+  // MARK: UICollectionViewDataSource
+  override func collectionView(collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
+    return images.count
+  }
+
+  override func collectionView(collectionView: UICollectionView, cellForItemAtIndexPath indexPath: NSIndexPath) -> UICollectionViewCell {
+
+    let cell = collectionView.dequeueReusableCellWithReuseIdentifier("Cell", forIndexPath: indexPath) as! Cell
+    let image = images[indexPath.item]
+    cell.configure(image: image)
+
+    return cell
+  }
+
+  // MAKR: UICollectionViewDelegateFlowLayout
+  func collectionView(collectionView: UICollectionView, layout
+    collectionViewLayout: UICollectionViewLayout,
+    sizeForItemAtIndexPath indexPath: NSIndexPath) -> CGSize {
+
+      let size = CGRectGetWidth(collectionView.bounds)
```

Comment:
Agreed, accessing the property directly makes it more readable in my opinion :sunglasses: